File: Nilai.py
x=int(input("Masukkan nilai : "))

if x>60 and x<=100:
    if (x>80 and x<=100):
        print("Nilai anda A")
    else:
        print("Nilai anda B")
    print("Keterangan : Lulus")
else:
    print("Nilai anda C")
    print("Ketarangan : Tidak Lulus")

File: Nilai.cpp
#include <iostream>
using namespace std;

int main () {
    int x = 0;
    
    cout<<" Masukkan Nilai : ";
    cin>>x;
    if (x>60 && x<=100) {
        if (x>80 && x<=100)
            cout << "Nilai anda A"<<endl;
        else
            cout<< "Nilai anda B"<<endl;
        cout << "Keterangan : Lulus";
    }else {
        cout << "Nilai anda C"<<endl;
        cout<< "Keterangan : Tidak lulus";
    }
     return 0;       
}